### DISCUSSION TITLE
I don't get it

### DISCUSSION THREAD

**Discussion by npearson72** [storybookjs/builder-vite#494] [General]
This project purports to use Vite, and perhaps in some way it does, but when I follow the instructions on a new Vite project, and then run `npm run storybook`, I'm seeing:

```
info @storybook/react v6.5.10
info
info => Loading presets

info => Ignoring cached manager due to change in manager config
ℹ ｢wdm｣: wait until bundle finished:
ℹ ｢wdm｣: Hash: ec535b9c1eefe51092a2
Version: webpack 4.46.0
Time: 5890ms
Built at: 09/08/2022 7:35:19 PM
                         Asset      Size        Chunks                    Chunk Names
           0.manager.bundle.js   287 KiB             0  [emitted]  [big]
           1.manager.bundle.js   128 KiB             1  [emitted]
           2.manager.bundle.js   892 KiB             2  [emitted]  [big]
           3.manager.bundle.js  83.9 KiB             3  [emitted]
           4.manager.bundle.js  15.8 KiB             4  [emitted]
           5.manager.bundle.js   295 KiB             5  [emitted]  [big]
           6.manager.bundle.js  1.36 KiB             6  [emitted]
                    index.html  5.64 KiB                [emitted]
        main.manager.bundle.js  4.58 KiB          main  [emitted]         main
runtime~main.manager.bundle.js  8.95 KiB  runtime~main  [emitted]         runtime~main
vendors~main.manager.bundle.js  4.73 MiB  vendors~main  [emitted]  [big]  vendors~main
...
```

Which is obviously webpack (and old version to boot), and not Vite.

Sorry, but what am I missing?

**Comment by IanVS** [comment]
Storybook has two parts, a manager (which is the storybook UI: sidebar, addons, toolbar), and the preview (the canvas where your components are rendered).  When you use the Vite builder, it replaces Webpack inside the preview with Vite, so that your components are rendered using Vite.  The manager itself is still built using Webpack 4 currently.  In Storybook 7, Webpack won't be installed at all if you're using a Vite framework.  The manager is now being pre-bundled with esbuild.

If you'd like to try out version 7, you can run `npx sb@future init` in a new project, or `npx sb@future upgrade --prerelease` if storybook is already installed.  It's still in alpha, so I recommend reading, or at least skimming, the migration notes: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-65x-to-700

**Reply by npearson72** [reply]
Excellent. Thank you.

Any ballpark ETA on 7 as final release? (Sorry, had to ask)

**Reply by IanVS** [reply]
I haven't heard any solid timeframes yet.  Last I heard was "end of this year or the beginning of the next."

**Reply by npearson72** [reply]
Thank you again!
